src/components/tabs/_macro-options.md

```diff
@@ -2,6 +2,7 @@
 | ------------------ | --------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
 | title              | string          | true     | The `h2` level heading for the tabs `section` element required to give context for screen readers. Visually hidden when tabs are visible. |
 | titleTag           | string          | false    | The HTML heading tag to wrap the title text in for it’s correct semantic order on the page. Will default to an `h2`                       |
+| titleTagClass      | string          | false    | Additional class to be added to titleTag                                                                                                  |
```

Comment:
Think we should rename `titleTagClass` to something like `titleClasses` or something because it isn't the tag the classes are being set on so I feel its more descriptive and fits more with what has been used elsewhere in the DS